src/reference/config/bindings/binding-kafka-proxy.md

```diff
@@ -0,0 +1,136 @@
+---
+shortTitle: kafka-proxy 
+description: Zilla runtime kafka-proxy binding
+icon: plus
+category:
+  - Binding
+tag:
+  - Proxy
+---
+
+# kafka-proxy Binding
+
+Zilla runtime kafka-proxy binding.
+
+```yaml {2}
+kafka_proxy0:
+  type: kafka-proxy
+  kind: proxy
+  options:
+    external:
+      host: kafka-#.external.net
+      port: 9093
+    internal:
+      host: b-#.kafka.internal.net
+      port: 9094
+  exit: tls_client0
+```
+
+## Summary
+
+Defines a binding with `kafka-proxy`  support, with `proxy` behavior.
+
+## Configuration
+
+:::: note Properties
+
+- [kind\*](#kind)
+- [options](#options)
+- [options.external](#options-external)
+  - [external\[\].host](#external-host)
+  - [external\[\].port](#external-port)
+- [options.internal](#options-internal)
+  - [internal\[\].host](#internal-host)
+  - [internal\[\].port](#internal-port)
```

Comment:
if options.external and options.internal are just objects and not lists of objects then we don't need the square brackets `\[\]`